ci/travis/36-pd024.yaml

```diff
@@ -15,15 +15,15 @@ dependencies:
   #- codecov
   # optional
   - rtree
-  - matplotlib==2.0.2
+  - matplotlib
   - descartes
   #- geopy
   - SQLalchemy
   - libspatialite
   - pyarrow
   - pip:
-    - pyproj
-    - codecov
-    - geopy
-    - mapclassify>=2.2.0
-    - git+https://github.com/pygeos/pygeos.git
+      - pyproj
```

Comment:
Maybe put here the `pyproj==2.3.1` pin that was in the deleted `36-pd023.yaml` file?